### ISSUE TITLE
Foramen ovale aneurysm

### ISSUE THREAD

**Issue by patt1morgan** [obophenotype/human-phenotype-ontology#7028]
**HPO term**
Atrial septal dilatation HP:0011995
**Synonym to be added**
Foramen ovale aneurysm, Redundant septum primum flap  PMID 9284005

**broad/exact/narrow/plural/abbreviation ?**

**Comment by pnrobinson** [comment]
@patt1morgan I think the existing term [Atrial septal dilatation HP:0011995](https://hpo.jax.org/app/browse/term/HP:0011995) refers to a postnatal finding and not Foramen ovale aneurysm. I will make a nerw term called Foramen ovale aneurysm and add these other labels as synonyms.

**Comment by pnrobinson** [comment]
Foramen ovale aneurysm also known as atrial septal aneurysm refers to an abnormal protrusion of the interatrial septum resulting in decreased left atrial volume.

On color Doppler there is reduced left ventricular inflow.

Foramen ovale aneurysm (FOA) or atrial septal aneurysm is abnormal redundancy of the atrial septum primum, with bulging of the septum by at least half the width of the atrial chamber, or by at least 10 mm beyond the level of the atrial septum.
Comment: The reported incidence in preterm and term infants is 11.1% and 7.1%, respectively.24 Postnatally, most atrial septal aneurysms resolve spontaneously as the pressure in the left atrium increases and compresses the septum against
the foramen ovale. However, atrial septal aneurysms may be associated with other cardiac anomalies including atrial septum defect, patent foramen ovale, and mitral valve prolapse. In the absence of other cardiac anomalies or arrhythmia, the prognosis for FOA is excellent. See figure 7 of PMID:30601441.

**Comment by pnrobinson** [comment]
adding term
